module/zfs/zfs_ioctl.c

```diff
@@ -4054,8 +4054,8 @@ static boolean_t zfs_ioc_recv_inject_err;
 #endif
 
 /*
- * On failure the 'errors' nvlist may be allocated and will contain a
- * descriptions of the failures.  It's the callers responsibilty to free.
+ * nvlist 'errors' is always allocated and will contain a descriptions of the
```

Comment:
s/a descriptions/descriptions/

should it also have "if any" at the end to make it clearer it can be empty?